### ISSUE TITLE
AttributeError: type object 'skia.GrContext' has no attribute 'MakeGL'

### ISSUE THREAD

**Issue by moefear85** [kyamagu/skia-python#185]
**Describe the bug**
I installed glfw, pyopengl, and the mesa glx and mesa dri packages etc. But no example from the documentation runs. I always get:

> AttributeError: type object 'skia.GrContext' has no attribute 'MakeGL'

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.10
 - Python: 3.10.7
 - skia-python version: 87.5

**Comment by dankope** [comment]
Just had similar problem, it was moved to `GrDirectContext`.

It has been moved to `skia.GrDirectContext.MakeGL()`. I found this by searching the api docs [here](https://kyamagu.github.io/skia-python/reference/skia.html). Hope this helps!

**Comment by moefear85** [comment]
no it doesn't, but I'm using Flutter/Flet instead. much simpler to set up and run. Hope this helps others who just want anti-aliased accelerated as-simple-as-possible gui+canvas toolkit for python.

**Comment by dankope** [comment]
> no it doesn't
Could you provide the code that is not working?

**Comment by HinTak** [comment]
The api docs is not a good guide... fact is, the two got merged - everything that used to belong to GrContext got merged into GrDirectContext in m88.

**Comment by HinTak** [comment]
Okay, even I got confused... the authoritative version is the test code (which is versioned, and also run per commit and before release): 
https://github.com/kyamagu/skia-python/blob/fbca09d0d55c9019dd2b864780ce000d533ca341/tests/conftest.py#L57

@emesare was essentially right, @moefear85 was wrong at 87.5. Anyway, it is a bit confusing as they got merged at m88, so if you are looking at upstream's API or other skia-bindings, you'd get the wrong idea. Anyway, in m1xx, one of them is aliased to the other, and both works.
